Marlin/stepper.cpp

```diff
@@ -601,14 +639,28 @@ void Stepper::init() {
   TCNT1 = 0;
   ENABLE_STEPPER_DRIVER_INTERRUPT();
 
-  #if ENABLED(ADVANCE)
+  #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
+
+    #if ENABLED(LIN_ADVANCE)
+
+      for (int i = 0; i < EXTRUDERS; i++) {
```

Comment:
I didn't want to completely drop the old `ADVANCE` feature because someone out there might find it charming and want to make it work better. I think it's probably a Good Thing™ to incorporate all the extant advance algorithms, because it makes a great reference to have the code _in-situ_ and where they can be compared and refined over time. In the longer run, we will probably end up with a nice unified algorithm.

If the old `ADVANCE` feature is irredeemably broken, then we may as well drop it, but I would like to understand it better (and what the original developer was thinking) before excising it completely.